src/mumble/Settings.cpp

```diff
@@ -332,6 +334,9 @@ Settings::Settings() {
 	qsTxAudioCueOn  = cqsDefaultPushClickOn;
 	qsTxAudioCueOff = cqsDefaultPushClickOff;
 
+	bTxMuteCue  = false;
```

Comment:
```suggestion
	bTxMuteCue  = true;
```
I think we can default to enabling this feature as it seems very handy. Therefore I think most folks like being warned about speaking when being muted and the ones that don't, can then go and turn it off for themselves.

Reply:
I did that, thanks. One case I could imagine where it's annoying is if you are passively listening (e.g. on the c3 lingo server). However in interactive cases it seems more useful. I also guess we don't have any infrastructure to easily toggle the default later, right? Then maybe it's better to start out this way and maybe offer a button in the top bar if that's something that needs toggling more often?

Reply:
Wdym "changing it later"? We can change it in any release if we wish to do so.

> maybe offer a button in the top bar if that's something that needs toggling more often?

Yeah that sounds good

Reply:
I guess I sort of assumed that any original default would be written into the local user config. If we only persist differences, that'd be easy. ;-)

Do we need the button in this rev of the change or can we wait for beta feedback?